File: Actividad1/Php/Libro.php
<?php
class Libro {
    public $titulo;
    public $autor;

    public function __construct($titulo, $autor) {
        $this->titulo = $titulo;
        $this->autor = $autor;
    }

    public function mostrarInfo() {
        return "Libro: {$this->titulo}, Autor: {$this->autor}";
    }
}

$libro = new Libro("Cien años de soledad", "Gabriel García Márquez");
echo $libro->mostrarInfo();
?>

File: Actividad1/Python/Libro.py
class Libro:
    def __init__(self, titulo, autor):
        self.titulo = titulo
        self.autor = autor

    def mostrar_info(self):
        return f"Libro: {self.titulo}, Autor: {self.autor}"

libro = Libro("El Principito", "Antoine de Saint-Exupéry")
print(libro.mostrar_info())